test/paginator_test.exs

```diff
@@ -590,12 +616,12 @@ defmodule PaginatorTest do
     p1 = insert(:payment, customer: c2, charged_at: days_ago(11))
     p2 = insert(:payment, customer: c2, charged_at: days_ago(6))
     p3 = insert(:payment, customer: c2, charged_at: days_ago(8))
-    p4 = insert(:payment, customer: c2, charged_at: days_ago(12))
+    p4 = insert(:payment, customer: c2, amount: 2, charged_at: days_ago(12))
```

Comment:
Is this change still needed?

Reply:
Kinda...but only in that it will ensure the test on line 288 fails if any pre-existing query filtering is ever broken

Reply:
Makes sense.